src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -200,6 +201,8 @@
     @JsonProperty("PidsLimit")
     private Long pidsLimit;
 
+    @JsonProperty("Tmpfs")
+    private Map<String, String> tmpFs;
```

Comment:
Could you check starting from what api version it appeared ?

Reply:
Docker 1.10, API version 1.22.
I just pushed a fixup.